delivery/src/main/res/layout-land/fragment_delivery_detail.xml

```diff
@@ -52,6 +52,40 @@
             android:paddingTop="10dp"
             android:paddingBottom="90dp">
 
+            <TextView
+                android:id="@+id/autodecline_countdown_info"
+                style="@style/DeliveryItemDetailCountdown"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
```

Comment:
otočil bych wrap šířky, popisek pružnej a zalamuje se, hodnota času na jeden řídek bez zalomení. Bude se to líp číst.

Reply:
stejně tak u portrait layoutu